### ISSUE TITLE
PPE-Untested/doesn't work

### ISSUE THREAD

**Issue by jentron** [jentron/Blender-PT2#7]
Poser Prop Exporter is untested and assumed not to work

**Comment by dlorre** [comment]
Hey I am interested by this because I own Poser Pro 2010 and need to import rigged meshes from Blender. 

I have fiddled with it a bit and it seems that you can no longer import modules, you have to import classes instead.

So, to get PPI12 working you can do:

    
    from . PPI12 import *

This video can also help: https://www.youtube.com/watch?v=uahfuypQQ04

If I can make a proper setup and if you continue to manage this project I will send a pull request. Thanks for your work.

Dominique Lorre

**Comment by jentron** [comment]
- The prop exporter is working for me (at least, its loading the GUI) without the changes to import.  Is the GUI not loading for you?
- There was a bug where it was looking in the wrong place for face data. I just fixed  that, so it is now exporting props.
- There is a case-sensitivity bug in the export, the pp2 has :Runtime:geometries: instead of :Runtime:Geometries: 
- UV's don't work

**Comment by dlorre** [comment]
Hey I am using Blender 2.81 on Windows, and I have only the importer working. I don't know about the space issue, I just installed VSCode so I don't know if it uses tabs or spaces, it should adapt to the source code normally. Good luck with this project.

**Comment by jentron** [comment]
I'm interested in your feedback on the project, and look forward to more pull requests in the future. Hopefully I'll do a better job merging them next time!

**Comment by dlorre** [comment]
Yes, I will look, but in the meantime I have found an exporter for Blender to DAZ which might do the job for me: https://sites.google.com/site/millighostmix/home/export_dsf_prop

Are you using VSCode?

**Comment by jentron** [comment]
No, generally just Notepad++, although I use Spyder for ipython stuff

**Comment by dlorre** [comment]
Ah I see, with VSCode you can test the addon directly by pressing Alt+P, it's explained in the video above.
